tracing-journald/src/lib.rs

```diff
@@ -345,8 +337,8 @@ impl Visit for EventVisitor<'_> {
     }
 }
 
-fn put_metadata(buf: &mut Vec<u8>, meta: &Metadata, span: Option<usize>) {
-    if span.is_none() {
+fn put_metadata(buf: &mut Vec<u8>, meta: &Metadata, span: bool) {
+    if !span {
```

Comment:
I think it would make more sense to move the `PRIORITY` field into a separate call (or just hardcode it directly in `on_event`).